tools/pd-ctl/pdctl/command/keyspace_group_command.go

```diff
@@ -47,6 +50,33 @@ func newSplitKeyspaceGroupCommand() *cobra.Command {
 	return r
 }
 
+func newFinishSplitKeyspaceGroupCommand() *cobra.Command {
```

Comment:
Will manual finish split/merge introduce side effects?

Reply:
change the finish commands to the hidden commands